tests/artifacts/javascript/on-chain-state-mgmt-zkapp-ui.js

```diff
@@ -96,7 +93,13 @@ updateButton.addEventListener('click', async (event) => {
       zkAppInstance.sign(zkAppPrivateKey);
```

Comment:
this causes it to create no proof

Reply:
Thank you!
What was the driver behind this decision (if sign present no proofs will be generated)?
Is it documented somewhere?

Reply:
not documented, but planned to be!
sign means a signature is to be used as authorization. 'signature' and 'proof'  are two mutually exclusive forms of authorization. you can't have both.

Reply:
Got it, thank you!